src/puppetlabs/jdbc_util/core.clj

```diff
@@ -165,3 +165,22 @@
       first
       :count
       pos?))
+
+(defn get-sequence-name
+  "Returns the name of the sequence associated with a column, or nil if there is
+  no sequence."
+  [db table column]
+  (-> (jdbc/query db ["SELECT pg_get_serial_sequence(?, ?)" table column])
+      first
+      :pg_get_serial_sequence))
+
+(defn reconcile-sequence!
```

Comment:
Yeah, I guess it's fine. the only real issue would be if the sequence gets out of whack in a way you care to observe, or if it's somehow a special sequence. But that kind of thing tends to occur when using sequences in some kind of replication system, which we're of course specifically avoiding here. So it should be fine.